mods/website/web/il8n/homepage.json

```diff
@@ -0,0 +1,60 @@
+    "peer2peer": {"en": "Peer-to-Peer Web3 Toolkit", "cn": "Web 3.0 PKI基础架构"},
+    "peer2peer_desc": {"en": "Libraries that support everything from QR-generation and scanning to ZKP-based gaming functions", "cn": "Saito去中心化第三方可信的区块链技术，可高效处理数据加密及认证"},
+    "app_deployment": {"en": "App Deployment", "cn": "开放式基础设施"},
+    "app_deployment_desc": {"en": "Open AppStore that lets developers build and deploy apps without the need to run any infrastructure", "cn": "Saito的底层架构可保证其它区块链技术以去中心化的方式在Saito上顺利运行"},
+    "enterprise_pki_support": {"en": "Enterprise PKI Support", "cn": "链上部署"},
+    "enterprise_pki_support_desc": {"en": "A scalable PKI network layer designed to handle enterprise-level key-exchange and encryption requirements", "cn": "通过点对点方式在链上分配DApp（有POC演示）"},
+    "vibrant_community": {"en": "Vibrant Community", "cn": "适度经济"},
+    "vibrant_community_desc": {"en": "A growing community of users interested in and excited about the applications want to build...", "cn": "通过简洁的经济原理设计出可扩展的吞吐规模"},
+    "simple_web3_dev": {"en": "Simple Web3 Development", "cn": "Web3简易开发"},
+    "truly_p2p": {"en": "Truly Peer\nto Peer", "cn": "真正的点对点"},
+    "scalable_onchain": {"en": "Scalable\nOnchain\nData", "cn": "可扩展的\n链上数据"},
+    "browser_app_without_metamask": {"en": "Browser\nApplications\nWithout\nMetamask", "cn": "浏览器\n应用无需使用\nMetamask钱包"},
+    "all_with_know_love": {"en": "All with the\ntools you know\nand love", "cn": "无数你熟悉\n和喜爱的工具"},
+    "transactions_perday": {"en": "Transactions/Day and Growing", "cn": "日交易量持续增长"},
+    "10_mil": {"en": "10+ Million", "cn": "800多万笔"},
+    "transactions": {"en": "Transactions", "cn": "交易"},
+    "30_popular": {"en": "30+ Popular", "cn": "30+热门"},
+    "apps_and_modules": {"en": "Applications and Modules", "cn": "应用和模块"},
+    "roadmap": {"en": "Roadmap", "cn": "路线图"},
+    "arcade_comm_growth": {"en": "Arcade and community growth", "cn": "商城和社区发展"},
+    "new_game_releases_arcade": {"en": "New Game Releases on Arcade", "cn": "2020商城游戏上新"},
+    "team_expansion": {"en": "Team expansion", "cn": "团队扩张"},
+    "expansion_erp": {"en": "Expansion of ERP/Logistics software", "cn": "扩大ERP和物流管理系统"},
+    "soc_trad_marketing": {"en": "Social and traditional marketing", "cn": "社会营销和传统营销"},
+    "onboard_3rd_dev": {"en": "Onboarding 3rd Party Developers", "cn": "第三方开发者加入"},
+    "fundraising_discount_tranches": {"en": "Fundraising through discount tranches", "cn": "开放私募折价认购"},
+    "legals_structure_public_sale": {"en": "Legals and Structure for Public Sale", "cn": "确立公售结构及合法性"},
+    "hardening_security": {"en": "Hardening and Security Work", "cn": "2021系统强化及安全"},
+    "our_supporters": {"en": "Our Supporters", "cn": "支持"},
+    "copyright": {"en": "© Proclus Technologies. All Rights Reserved", "cn": "© Proclus Technologies. 版权所有"},
+    "arcade": {"en": "Arcade", "cn": "商城"},
+    "developers": {"en": "Developers", "cn": "开发者"},
+    "community": {"en": "Community", "cn": "社区"},
+    "": {"en": "", "cn": ""}
```

Comment:
remove this line, i think.

Are these keys the id of the element?

Reply:
oh the last line? yeah I'll delete it